netbox/dcim/views.py

```diff
@@ -2122,6 +2122,17 @@ class DeviceBulkRenameView(generic.BulkRenameView):
     filterset = filtersets.DeviceFilterSet
     table = tables.DeviceTable
 
+    def _rename_objects(self, form, selected_objects):
```

Comment:
Private view methods should not be overridden. Additionally, it's not clear why this logic would apply to devices but not other objects being renamed in bulk.